.github/workflows/e2e.yaml

```diff
@@ -71,17 +73,25 @@ jobs:
         name: e2e-test-logs
         path: ./e2e_logs/
         retention-days: 15
+    
+    - name: Archive Ginkgo logs
+      uses: actions/upload-artifact@v4
+      with:
+        name: ginkgo-logs
+        path: ginkgo.json
+        retention-days: 15
 
-    - name: Send Slack alert notification
-      id: slack
-      if: false
+    - name: Post text to a Slack channel
+      if: ${{ failure() }}
       uses: slackapi/slack-github-action@v2.0.0
       env:
         SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
         SUMMARY_URL: https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}
       with:
-        channel-id: ${{ secrets.SLACK_CHANNEL_ID }}
-        slack-message: |
-          :x: On repository ${{ github.repository }} the Workflow *${{ github.workflow }}* has failed.
+        method: chat.postMessage
+        token: ${{ secrets.SLACK_BOT_TOKEN }}
+        payload: |
+          channel: ${{ secrets.SLACK_CHANNEL_ID }}
+          text: ":x: On repository ${{ github.repository }} the Workflow *${{ github.workflow }}* has failed.
 
-          Details: ${{ env.SUMMARY_URL }}
+          Details: ${{ env.SUMMARY_URL }}"
```

Comment:
can we indent this? Initially, I thought `Details:` was another yaml field

Reply:
it is indented